draft-ietf-rats-architecture.md

```diff
@@ -360,13 +360,12 @@ The Verifier uses the Evidence, and any Endorsements from Endorsers,
 by applying an Appraisal Policy for Evidence to assess the trustworthiness of the Attester,
 and generates Attestation Results for use by Relying Parties.  The Appraisal Policy for Evidence
 might be obtained from an Endorser along with the Endorsements, or might be obtained
-via some other mechanism such as being configured in the Verifier by an
-administrator.
+via some other mechanism such as being configured in the Verifier by the Verifier Owner.
 
 The Relying Party uses Attestation Results by applying its own
 Appraisal Policy to make application-specific decisions such as authorization decisions.
-The Appraisal Policy for Attestation Results might, for example, be configured in the Relying Party
-by an administrator.
+The Appraisal Policy for Attestation Results is configured in the Relying Party
```

Comment:
I prefer ‘might be’ here, and configuration may be just one of the possible mechanisms. Suggest changing it back.